### PR TITLE
css/navitem-multiselect

### LINKED ISSUE #1916: convert css for navitem

### LINKED ISSUE #1917: convert css for navcookiebanner

### LINKED ISSUE #1918: convert css for navbrand

### LINKED ISSUE #1919: convert css for navbar

### LINKED ISSUE #1920: convert css for multiselect

### PR DESCRIPTION
**Pull request by EdwardIrby**
resolves #1916 
resolves #1917 
resolves #1918 
resolves #1919 
resolves #1920

### CODE REVIEW

File: packages/navbar/src/css/index.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
+.psds-navbar {;
```
Comment (jaketrent) [inline packages/navbar/src/css/index.css:1]:
```suggestion
.psds-navbar {
```

File: packages/navbar/src/css/index.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  display: flex;
+  align-items: center;
+  width: 100%;
+  height: 56px;
+  padding: var(--psLayoutSpacingXSmall);
```
Comment (jaketrent) [inline packages/navbar/src/css/index.css:6]: ps var problem again -- needs dashify

File: packages/navbrand/src/react/index.tsx [hunk 3]
```diff
@@ -82,9 +56,17 @@ const NavBrand = React.forwardRef<NavBrandElement, NavBrandProps>(
 export default NavBrand
 
 const Logo: React.FC = props => (
-  <div {...styles.logo(Boolean(props.children))} {...props} />
+  <div
+    className={classNames(
+      'psds-navbrand__logo',
+      Boolean(props.children) && 'psds-navbrand__logo--wordmark'
```
Comment (jaketrent) [inline packages/navbrand/src/react/index.tsx:62]: Not sure it's any safer, but `React.Children.count(props.children) > 0` comes to mind.

**Review by jaketrent — CHANGES_REQUESTED** [review]
Not sure how this happened. The camelCase psds core vars are incorrect. They need dashified. Maybe something weird with the version of the update script?? Anyway, storybook confirms that these have lots of visual issues. I think getting the vars fixed up is priority 1.

**Review by jaketrent — CHANGES_REQUESTED** [review]
Code good to my eye. 

There's something wrong with Multiselect. It's a 2-rower. Not so in master. I did a rebuild. Must be something going on in CSS? Anyone else seeing the same?

Screen Shot 2021-07-16 at 2 48 12 PM

**Review by sweeting — CHANGES_REQUESTED** [review]
Reviewed all visually and the only issue I noticed was the one [Jake mentioned](https://github.com/pluralsight/design-system/pull/1936#pullrequestreview-708750359) regarding the double height by default on the multiselect.

Jake created a [PR](https://github.com/pluralsight/design-system/pull/1940) that fixes that issue and I reviewed that and it looks correct.

**Comment by EdwardIrby** [comment]
Fixed the double row thing though it made some things clear about the Field component and styles. Let's chat about this tomorrow morning.

**Review by sweeting — APPROVED** [review]
Multiselect looks good to me now.

File: packages/multiselect/src/css/index.css [hunk 4]
```diff
@@ -77,9 +77,10 @@
 
 .psds-multi-select__input-container {
   margin: calc(4px / 2);
+  flex: 1;
 }
 
-.psds-multi-select__input {
+.psds-multi-select__input-container>.psds-field__input {
```
Comment (jaketrent) [inline packages/multiselect/src/css/index.css:83]: nit: I'd put spaces around the child combinator, stylistically treating it like other operators. I'm surprised prettier didn't do this. But then, maybe prettier isn't running on our css.

**Review by jaketrent — APPROVED** [review]

**Pull request merged**